papierowyRPG_API/Models/Character.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace papierowyRPG_API.Models
+{
+    public class Character
+    {
+        public int Id { get; set; }
+        public int Id_user { get; set; }
+        public int Id_game { get; set; }
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public string story { get; set; }
```

Comment:
For now it can be string but we should think about some way to store large strings, as story could be pretty big chunk of text